mlflow/server/js/src/experiment-tracking/components/ExperimentView.js

```diff
@@ -90,8 +91,12 @@ export class ExperimentView extends Component {
     this.getStartTimeColumnDisplayName = this.getStartTimeColumnDisplayName.bind(this);
     this.onHandleStartTimeDropdown = this.onHandleStartTimeDropdown.bind(this);
     this.handleDiffSwitchChange = this.handleDiffSwitchChange.bind(this);
+    const urlState = Utils.getSearchParamsFromUrl(this.props.location.search);
     const store = ExperimentView.getLocalStore(this.props.experiment.experiment_id);
-    const persistedState = new ExperimentViewPersistedState(store.loadComponentState());
+    const persistedState = new ExperimentViewPersistedState({
+      ...store.loadComponentState(),
+      ...urlState,
+    });
```

Comment:
I think it would help to parse the URL in experimentPage and sending the state down which can be used as default state for the current values that we have. 

Regarding the `getDerivedStateFromProps`, when the experiment changes, I don't think the constructor is called again. In that case, do we need the URL state to correctly update the `ExperimentViewPersistedState` correctly? We can also test this usecase by click the back button of the browser and seeing that the URL state changing, does that change the  `ExperimentViewPersistedState` correctly as well?